### PR TITLE
Add oc-certification artifacts and update makefile

### LINKED ISSUE #114: Add OC-Certified Artifacts
**What type of Issue/Feature is this?**
 /kind enhancement

### PR DESCRIPTION
**Pull request by asifdxtreme**
**What type of PR is this?**
 /kind enhancement

**What this PR does / why we need it**:
This PR adds the artifacts which was certified by redHat and updates the Makefile

**Which issue(s) this PR fixes**:
Fixes #114

**Test Report Added?**:
 /kind TESTED

### CODE REVIEW

File: config/openshift/rbac/vdo-controller-manager-metrics-service-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
```
Comment (jvrahav) [inline config/openshift/rbac/vdo-controller-manager-metrics-service-rbac.yaml:2]: this is the service definition not the rbac. can you please check this

File: config/openshift/csv/kustomization.yaml [hunk 2]
```diff
@@ -1,11 +0,0 @@
-# Adds namespace to all resources.
```
Comment (jvrahav) [inline config/openshift/csv/kustomization.yaml:1]: config/openshift/csv folder can be deleted. as we are not using the csv file from this location

File: artifacts/oc-certified/0.1.0/manifests/vdo-controller-manager-metrics-service-rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Service
```
Comment (jvrahav) [inline artifacts/oc-certified/0.1.0/manifests/vdo-controller-manager-metrics-service-rbac.yaml:3]: this is metrics-service. what are we naming it rbac

**Review by jvrahav — CHANGES_REQUESTED** [review]

File: config/openshift/csv/kustomization.yaml (hunk 2 above)
Comment (asifdxtreme) [reply config/openshift/csv/kustomization.yaml:1]: This has been deleted

File: artifacts/oc-certified/0.1.0/manifests/vdo-controller-manager-metrics-service-rbac.yaml (hunk 3 above)
Comment (asifdxtreme) [reply artifacts/oc-certified/0.1.0/manifests/vdo-controller-manager-metrics-service-rbac.yaml:3]: this is already merged in the certified operators project so we cannot modify it, we will update this in next release

File: config/openshift/rbac/vdo-controller-manager-metrics-service-rbac.yaml (hunk 1 above)
Comment (asifdxtreme) [reply config/openshift/rbac/vdo-controller-manager-metrics-service-rbac.yaml]: Update

**Review by jvrahav — APPROVED** [review]

**Review by ridaz — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**